spec/requests/api/v1/events/events_spec.rb

```diff
@@ -26,3 +26,57 @@
     )
   end
 end
+
+describe 'POST/v1/events' do
+  it 'saves the address, lat, lon, name, and started_at date' do
+    event = build :event
+    event_params = {
+      address: event.address,
+      ended_at: event.ended_at,
+      lat: event.lat,
+      lon: event.lon,
+      name: event.name,
+      started_at: event.started_at,
+      owner: {device_token: event.owner.device_token
+      }
+    }
+    post '/v1/events', { event: event_params }.to_json, 'Content-Type' => 'application/json'
+
+    response_event = Event.last
+    expect(response_json).to eq('id' => response_event.id)
+    expect(response_event.address).to eq event.address
+    expect(response_event.ended_at.to_i).to eq event.ended_at.to_i
+    expect(response_event.lat).to eq event.lat
+    expect(response_event.lon). to eq event.lon
+    expect(response_event.name).to eq event.name
+    expect(response_event.started_at.to_i).to eq event.started_at.to_i
+    expect(response_event.owner).to eq event.owner
+  end
+
+  it 'returns an error message when invalid' do
+      empty_params = {
+        address: nil,
+        ended_at: nil,
+        lat: nil,
+        lon: nil,
+        name: nil,
+        started_at: nil,
+        owner: {
+        device_token: nil
```

Comment:
Again, minor code style, either indent `device_token` or one-line it alongside `owner`.